docs/synapse/userguides/storm_ref_type_specific.ipynb

```diff
@@ -263,61 +581,233 @@
+    "\n",
+    "``ival`` types have their own optimizations in addition to those related to ``time`` types.\n",
+    "\n",
+    "Indexing\n",
+    "++++++++\n",
+    "\n",
+    "N/A\n",
+    "\n",
+    "Parsing\n",
+    "+++++++\n",
+    "\n",
+    "An ``ival`` type is typically specified as two comma-separated time values enclosed in parentheses. Alternately, an ``ival`` can be specified as a single time value with no parentheses (see **Insertion** below for ``ival`` behavior when specifying a single time value).\n",
+    "\n",
+    "Single or double quotes are required in accordance with the standard rules for :ref:`whitespace` and :ref:`literals`. For example:\n",
+    "\n",
+    "- ``[ .seen=(\"2017/03/24 12:13:27\", \"2017/08/05 17:23:46\") ]``\n",
+    "- ``[ +#sometag=(2018/09/15, \"+24 hours\") ]``\n",
+    "- ``[ .seen=2019/03/24 ]``\n",
+    "\n",
+    ".. NOTE::\n",
+    "  As of this writing, the Storm parser recognizes time formats as human-readable strings and will convert them to epoch millis. This means that currently, even though the values are strings, they do not need to be quoted unless they contain whitespace. However, in order for the runtime to better distinguish (and interpret) integers as integers and strings as strings, **future** versions will likely require enclosing all numeric string values (i.e., values that could be interpreted as integers) in quotes. Specifically, \"unformatted\" time strings such as ``20190218094713`` (for ``\"2019/02/18 09:47:13\"``) that could be misinterpreted as integers will need to be quoted. In anticipation of this change, it may be helpful to simply enclose all time strings (regardless of formatting or whitespace) in quotes.\n",
+    "\n",
+    "When entering an ``ival`` type, each time value can be input using most of the acceptable formats for :ref:`type-time` types, including explicit times, relative times, and the special values ``now`` and ``?``.\n",
+    "\n",
+    "``ival`` types also support relative times using ``+-`` format to represent both a positive and negative offset from a given point (i.e., ``\"+-1 hour\"``).\n",
+    "\n",
+    "When entering relative times in an ``ival`` type:\n",
+    "\n",
+    "- A relative time in the **first** (``<min>``) position is calculated relative to the **current time** (``now``).\n",
+    "- A relative time in the **second** (``<max>``) position is calculated relative to the **first** (``<min>``) time.\n",
+    "\n",
+    "For example:\n",
     "\n",
-    "Indexing\n",
-    "++++++++\n",
+    "- ``.seen=(\"+-1 hour\")`` means from one hour before to one hour after the current time.\n",
+    "- ``.seen=(2018/12/01, \"+1 day\")`` means from 12:00 AM December 1, 2018 to 12:00 AM December 2, 2018.\n",
+    "- ``.seen=(2018/12/01, \"-1 day\")`` means from 12:00 AM November 30, 2018 to 12:00 AM December 1, 2018.\n",
+    "- ``.seen=(\"-30 minutes\", \"+1 hour\")`` means from 30 minutes ago to 30 minutes from now.\n",
```

Comment:
Did you mean `to 1 hour from now`?

Reply:
Newp - because the "+1 hour" is calculated from the initial "-30 minutes" :-)

Reply:
My bad, read that comment as referring to a different example. Fixing.